dev-packages/request/src/node-request-service.ts

```diff
@@ -98,7 +98,7 @@ export class NodeRequestService implements RequestService {
                 path: endpoint.pathname + endpoint.search,
                 method: options.type || 'GET',
                 headers: options.headers,
-                agent: options.agent,
+                agent: options.agent as unknown as http.Agent,
```

Comment:
Weird, but I can't build Theia without this. Looks like the node types were updated or something.